client/src/components/InputField.jsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+
+function InputField(props) {
+  const {
+    name, value, onChange, label, checked, type,
+  } = props;
+  const id = `${name}-${value}`;
+  return (
+    <label htmlFor={name}>
```

Comment:
Instead of using 'name', using 'id' so the buttons will be selected even clicking on the texts.

`<label htmlFor={id}>`